extension/content.css

```diff
@@ -478,3 +478,8 @@ hidden content area created to increase hover target
 	padding-top: 0 !important;
 	border-top: none !important;
 }
+
+.comment-full-name {
+	font-weight: normal;
+	margin-left: 1%;
```

Comment:
% value was just a force of habit - `margin-left` is to match the UI you mentioned you preferred in #250 - without it, it'll look like this:

![image](https://cloud.githubusercontent.com/assets/5233399/16155961/73f5060c-3477-11e6-8e11-b7f75509f110.png)